zetaclient/utils.go

```diff
@@ -170,6 +170,32 @@ func (t *DynamicTicker) Stop() {
 	t.impl.Stop()
 }
 
+// CheckEvmTxLog checks the basics of an EVM tx log
+func (ob *EVMChainClient) CheckEvmTxLog(vLog *ethtypes.Log, wantAddress ethcommon.Address, wantHash string, wantTopics int) error {
```

Comment:
Can we add tests for this method?

Reply:
same [commit](https://github.com/zeta-chain/node/pull/1539/commits/cb257f672246dee79554c565f711dec5b3ab1345)